### PR TITLE
[16.0][IMP] product_packaging_container_deposit: Add more context for update order container deposit quantity

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

File: product_packaging_container_deposit/models/container_deposit_order_mixin.py [hunk 1]
```diff
@@ -86,7 +86,9 @@ def update_order_container_deposit_quantity(self):
                         product, deposit_container_qties[product][0]
                     )
                     values_lst.append(Command.create(values))
-            order.write({self._get_order_line_field(): values_lst})
+            order.with_context(update_order_container_deposit_quantity=True).write(
```
Comment (simahawk) [inline product_packaging_container_deposit/models/container_deposit_order_mixin.py]: there's already the key `skip_update_container_deposit` at line 36: do you really need this?

Comment (nilshamerlinck) [reply product_packaging_container_deposit/models/container_deposit_order_mixin.py]: not exactly the same purpose:
- skip_update_container_deposit: avoid update on copy()
- update_order_container_deposit_quantity: flag to circumvent the fields protection

Comment (simahawk) [reply product_packaging_container_deposit/models/container_deposit_order_mixin.py]: Fair enough, but if you keep it, can you move it to the same place of the one above to not change ctx on _every_ record?

Comment (QuocDuong1306) [reply product_packaging_container_deposit/models/container_deposit_order_mixin.py]: Thank @nilshamerlinck and @simahawk , I updated

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@QuocDuong1306 the msg could be shorter and more explicit: `[IMP] product_packaging_container_deposit: add ctx key for qty update`

**Comment by QuocDuong1306** [comment]
Thank @simahawk , done here

**Pull request merged**